src/ray/raylet/scheduling/policy/node_affinity_scheduling_policy.cc

```diff
@@ -24,7 +24,13 @@ scheduling::NodeID NodeAffinitySchedulingPolicy::Schedule(
   scheduling::NodeID target_node_id = scheduling::NodeID(options.node_affinity_node_id);
   if (nodes_.contains(target_node_id) && is_node_alive_(target_node_id) &&
       nodes_.at(target_node_id).GetLocalView().IsFeasible(resource_request)) {
```

Comment:
The logic looks good, but I feel we could clean up the if statements a little. The nested if / return is a bit hard to parse.

Reply:
ditto :)